README.md

```diff
@@ -8,7 +8,7 @@
 - **Owner**: @emmanuelmathot
 
 This document explains the Order Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
-This extension allows assets ordering management within STAC specification.
+This extension allows assets ordering management within STAC specification.  In the context of this extension, the term "order" refers to a request for data to be deliveried, served, or otherwise made available.
```

Comment:
```suggestion
This extension allows assets ordering management within STAC specification.
In the context of this extension, the term "order" refers to a request for data to be delivered, served, or otherwise made available (e.g. after a purchase).
```